frontend/styles/components/dashboard-element/dashboard-welcome.scss

```diff
@@ -1,27 +1,67 @@
 .c-dashboard-welcome {
-  padding: 20px 25px;
+  padding: 40px 25px;
   .title {
     margin: 0 0 45px 0;
+    font-family: Merriweather;
+    font-size: 38px !important;
+    font-weight: 300 !important;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: 0.79;
+    letter-spacing: -1px;
+    text-align: center;
+    color: #34444c;
   }
 
   .dashboard-welcome-card-container {
     display: flex;
-    margin: 0 0 30px 0;
+    margin: 0 0 60px 15px;
   }
 
   .dashboard-welcome-card {
-    background-color: darken($light-gray, 7%);
+    font-family: Merriweather;
```

Comment:
Let's use the sass variables when setting the `font-family` and `font-weight` values. For other things such as `font-size` and `margin` and `padding` we aren't really using the sass variables anymore. I don't really expected you to know this so let's discuss it later.